source/interface/azure_iot_flash_platform.h

```diff
@@ -17,6 +17,8 @@
 
 AzureIoTResult_t AzureIoTPlatform_Init( AzureADUImage_t * const pxAduImage );
 
+uint32_t AzureIoTPlatform_GetFlashBankSize();
```

Comment:
Nit: naming - should it include "active/passive" or "OTA"? The device will likely have at least 3 banks: boot, bank1, bank2.

Reply:
Hmm, I would lean away from including OTA since none of the other flash platform functions have it, but it would be good to specify that this is the size of a single boot bank. maybe AzureIoTPlatform_GetSingleFlashBankSize or AzureIoTPlatform_GetFlashBootBankSize? I'll think a bit more on coming up with a concise and clear name

Reply:
Updated the name - let me know what you think! I leaned towards more descriptive since it's a fairly specific number we're looking for

Reply:
LGTM! Thank you!